query-engine/connector-test-kit-rs/query-engine-tests/tests/new/ref_actions/on_delete/set_default.rs

```diff
@@ -84,8 +84,8 @@ mod one2one_req {
     }
 
     /// Deleting the parent with no default for SetDefault fails.
-    /// Only postgres allows setting no default for a SetDefault FK.
-    #[connector_test(schema(required_without_default), only(Postgres))]
+    /// Only postgres (except CockroachDB) allows setting no default for a SetDefault FK.
+    #[connector_test(schema(required_without_default), only(Postgres), exclude(Cockroach))]
```

Comment:
This should not work. Cockroach is not a connector tag, but a postgres connector tag version. The syntax you're looking for is `exclude(Postgres(Cockroach))`. If the code here actually compiles and runs, then this is a bug.

Reply:
`"cockroach" => Self::Postgres(PostgresConnectorTag::new(version)?),` this line from a previous PR is a hack that allows you to do it. This is incorrect and an oversight on my part. You have to decide: Is cockroach a postgres version or is a connector. Vitess for example is technically a MySQL version, but we chose to use a separate connector for it in the tests instead of a version.

I will allow this PR for now because it unblocks you, but this will be cleaned up soon.